lumicks/pylake/kymotracker/tests/test_greedy_algorithm.py

```diff
@@ -39,6 +39,15 @@ def test_kymotracker_subset_test_greedy(kymo_integration_test_data):
     np.testing.assert_allclose(tracks[0].sample_from_image(1), [40] * np.ones(10))
 
 
+def test_kymotracker_regression_test_subset_comes_up_empty(kymo_integration_test_data):
+    """Test whether we gracefully handle a kymotracker where the rect results in no lines. Before
+    this threw a cryptic error message."""
```

Comment:
Do we need the comment about what happened before?

Reply:
I added it because it's a regression and if it happens again, it might provide some information on what happened last time this came up. That said, I don't feel particularly attached to this part of the comment, so I can remove it if you think it's not helpful.

Reply:
I changed it to:
```python
"""Test whether we gracefully handle the case where the ROI results in no lines."""
```